state.go

```diff
@@ -0,0 +1,70 @@
+// +build linux
+
+package main
+
+import (
+	"os"
+	"time"
+
+	"encoding/json"
```

Comment:
Can you club this with the other standard library imports and in alphabetical order?